master/docs/relnotes/0.9.0b9.rst

```diff
@@ -0,0 +1,105 @@
+Release Notes for Buildbot 0.9.0b9
+==================================
+
+The following are the release notes for Buildbot 0.9.0b9
+This version was released on May 10, 2016.
+
+See :ref:`Upgrading to Nine` for a guide to upgrading from 0.8.x to 0.9.x
+
+Master
+------
+
+Features
+~~~~~~~~
+
+* new :bb:reporter:`GitLabStatusPush` to report builds results to GitLab.
+
+* ``buildbot stop`` now waits for complete buildmaster stop by default.
+
+* New ``--no-wait`` argument for ``buildbot stop`` which allows not to wait for complete master shutdown.
+
+* Builder page is now sorted by builder name
+
+* LogViewer page now supports ansi color codes, and is displayed white on black.
+
+* This is the last version of buildbot nine which supports python 2.6 for the master.
```

Comment:
well this one still features python 2.6, as we did not yet merged your patches. Maybe I should put it in an additional Note section.